modules/imaging_browser/jsx/imagingBrowserIndex.js

```diff
@@ -0,0 +1,191 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Loader from 'Loader';
+import FilterableDataTable from 'FilterableDataTable';
+
+class ImagingBrowserIndex extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            data: {},
+            error: false,
+            isLoaded: false,
+        };
+
+        this.fetchData = this.fetchData.bind(this);
+        this.formatColumn = this.formatColumn.bind(this);
```

Comment:
i think this line is unnecessary since there's no use of `this` in formatColumn

Reply:
Removed!